third_party/tsl/tsl/profiler/lib/nvtx_utils.cc

```diff
@@ -33,6 +56,20 @@ ProfilerDomainHandle DefaultProfilerDomain() {
   return domain;
 }
 
+void NameCurrentThread(const std::string& thread_name) {
```

Comment:
Nit: convention is absl::string_view

Reply:
We need a C-style string for the NVTX API, so this would require an extra copy - I believe this is why the other `tsl::profiler` methods don't take `absl::string_view`, and I chose to be consistent here. I'll leave this for now but can change if you prefer.

(Of course, we don't often name devices/threads, so the cost of the copy is probably not relevant for these methods specifically)